drivers/hmis/app/graphql/types/hmis_schema/last_contact.rb

```diff
@@ -0,0 +1,16 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::LastContact < Types::BaseObject
+    skip_activity_log
+
+    field :date, GraphQL::Types::ISO8601Date, null: false
+    field :type, String, null: false
```

Comment:
If we drop the dynamic service name then this should be a fixed set of contact types that we can be represented as an enum, right?  If you think it shouldn't be an enum, I would give this field a different name like "description" or "name". `type` to me feels more like an enum field, rather than a string.

Reply:
Hm, yeah that seems reasonable! It can be an enum if we drop both dynamic service name and dynamic assessment name too

Reply:
Yeah I don't think we want the assessment name dynamic either, too much variability in length